pom.xml

```diff
@@ -11,6 +11,92 @@
     <properties>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <maven-surefire-plugin.suiteXmlFile>${project.build.testOutputDirectory}/testng/for.the.first.group.xml
```

Comment:
as I could see you have hardcoded default suite name.
By default should be specified **add and subtract maven profile**

Reply:
I added tag <activeByDefault> to the profile with id "for_the_first_group_plugin" which means add and subtract operations.